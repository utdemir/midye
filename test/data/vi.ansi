[?1000h[?1049h[>4;2m[?1h=[?2004h[1;12r[?12h[?12l[22;2t[22;1t[27m[23m[29m[m[H[2J[?25l[2;1H[94m~                                                                                                                                                                             [3;1H~                                                                                                                                                                             [4;1H~                                                                                                                                                                             [5;1H~                                                                                                                                                                             [6;1H~                                                                                                                                                                             [7;1H~                                                                                                                                                                             [8;1H~                                                                                                                                                                             [9;1H~                                                                                                                                                                             [10;1H~                                                                                                                                                                             [11;1H~                                                                                                                                                                             [m[12;157H0,0-1[9CAll[1;1H[?25h[?25l[12;147H^@[1;1H[12;147H  [1;1H[?25h[?25l[12;147H^M[1;1H[12;147H  [1;1H[?25h[?25l[12;147H^M[1;1H[12;147H  [1;1H[?25h[?25l[12;147Hi[1;1H[12;147H [1;1H[12;1H[1m-- INSERT --[m[12;157H[K[12;157H0,1[11CAll[1;1H[?25h[?25ld[12;157H1,2[1;2H[?25h[?25lo[12;159H3[1;3H[?25h[?25li[12;159H4[1;4H[?25h[?25ln[12;159H5[1;5H[?25h[?25lg[12;159H6[1;6H[?25h[?25l[12;159H7[1;7H[?25h[?25ls[12;159H8[1;8H[?25h[?25lt[12;159H9[1;9H[?25h[?25lu[12;159H10[1;10H[?25h[?25lf[12;160H1[1;11H[?25h[?25lf[12;160H2[1;12H[?25h[12;1H[K[1;11H[?25l[12;147H^[[1;11H[12;147H  [1;12H[12;157H1,11[10CAll[1;11H[?25h[?25l[12;147H:[1;11H[12;147H[K[12;1H:[?25hq![?25l[?1000l[?2004l[>4;m[23;2t[23;1t[12;1H[K[12;1H[?2004l[?1l>[?25h[>4;m[?1049l