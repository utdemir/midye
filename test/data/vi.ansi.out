TBSpecial (TSControlCharacter CC_ESC)
[?1000h
TBSpecial (TSControlCharacter CC_ESC)
[?1049h
TBSpecial (TSControlCharacter CC_ESC)
[>4;2m
TBSpecial (TSControlCharacter CC_ESC)
[?1h
TBSpecial (TSControlCharacter CC_ESC)
=
TBSpecial (TSControlCharacter CC_ESC)
[?2004h
TBSpecial (TSControlCharacter CC_ESC)
[1;12r
TBSpecial (TSControlCharacter CC_ESC)
[?12h
TBSpecial (TSControlCharacter CC_ESC)
[?12l
TBSpecial (TSControlCharacter CC_ESC)
[22;2t
TBSpecial (TSControlCharacter CC_ESC)
[22;1t
TBSpecial (TSTermSequenceCSI (TSC_SGR [27]))
TBSpecial (TSTermSequenceCSI (TSC_SGR [23]))
TBSpecial (TSTermSequenceCSI (TSC_SGR [29]))
TBSpecial (TSTermSequenceCSI (TSC_SGR []))
TBSpecial (TSControlCharacter CC_ESC)
[H
TBSpecial (TSControlCharacter CC_ESC)
[2J
TBSpecial (TSControlCharacter CC_ESC)
[?25l
TBSpecial (TSControlCharacter CC_ESC)
[2;1H
TBSpecial (TSTermSequenceCSI (TSC_SGR [94]))
~                                                                                                                                                                             
TBSpecial (TSControlCharacter CC_ESC)
[3;1H~                                                                                                                                                                             
TBSpecial (TSControlCharacter CC_ESC)
[4;1H~                                                                                                                                                                             
TBSpecial (TSControlCharacter CC_ESC)
[5;1H~                                                                                                                                                                             
TBSpecial (TSControlCharacter CC_ESC)
[6;1H~                                                                                                                                                                             
TBSpecial (TSControlCharacter CC_ESC)
[7;1H~                                                                                                                                                                             
TBSpecial (TSControlCharacter CC_ESC)
[8;1H~                                                                                                                                                                             
TBSpecial (TSControlCharacter CC_ESC)
[9;1H~                                                                                                                                                                             
TBSpecial (TSControlCharacter CC_ESC)
[10;1H~                                                                                                                                                                             
TBSpecial (TSControlCharacter CC_ESC)
[11;1H~                                                                                                                                                                             
TBSpecial (TSTermSequenceCSI (TSC_SGR []))
TBSpecial (TSControlCharacter CC_ESC)
[12;157H0,0-1
TBSpecial (TSControlCharacter CC_ESC)
[9CAll
TBSpecial (TSControlCharacter CC_ESC)
[1;1H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_BEL)
TBSpecial (TSControlCharacter CC_ESC)
[?25l
TBSpecial (TSControlCharacter CC_ESC)
[12;147H^@
TBSpecial (TSControlCharacter CC_ESC)
[1;1H
TBSpecial (TSControlCharacter CC_ESC)
[12;147H  
TBSpecial (TSControlCharacter CC_ESC)
[1;1H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25l
TBSpecial (TSControlCharacter CC_ESC)
[12;147H^M
TBSpecial (TSControlCharacter CC_ESC)
[1;1H
TBSpecial (TSControlCharacter CC_ESC)
[12;147H  
TBSpecial (TSControlCharacter CC_ESC)
[1;1H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25l
TBSpecial (TSControlCharacter CC_ESC)
[12;147H^M
TBSpecial (TSControlCharacter CC_ESC)
[1;1H
TBSpecial (TSControlCharacter CC_ESC)
[12;147H  
TBSpecial (TSControlCharacter CC_ESC)
[1;1H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25l
TBSpecial (TSControlCharacter CC_ESC)
[12;147Hi
TBSpecial (TSControlCharacter CC_ESC)
[1;1H
TBSpecial (TSControlCharacter CC_ESC)
[12;147H 
TBSpecial (TSControlCharacter CC_ESC)
[1;1H
TBSpecial (TSControlCharacter CC_ESC)
[12;1H
TBSpecial (TSTermSequenceCSI (TSC_SGR [1]))
-- INSERT --
TBSpecial (TSTermSequenceCSI (TSC_SGR []))
TBSpecial (TSControlCharacter CC_ESC)
[12;157H
TBSpecial (TSControlCharacter CC_ESC)
[K
TBSpecial (TSControlCharacter CC_ESC)
[12;157H0,1
TBSpecial (TSControlCharacter CC_ESC)
[11CAll
TBSpecial (TSControlCharacter CC_ESC)
[1;1H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25ld
TBSpecial (TSControlCharacter CC_ESC)
[12;157H1,2
TBSpecial (TSControlCharacter CC_ESC)
[1;2H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25lo
TBSpecial (TSControlCharacter CC_ESC)
[12;159H3
TBSpecial (TSControlCharacter CC_ESC)
[1;3H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25li
TBSpecial (TSControlCharacter CC_ESC)
[12;159H4
TBSpecial (TSControlCharacter CC_ESC)
[1;4H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25ln
TBSpecial (TSControlCharacter CC_ESC)
[12;159H5
TBSpecial (TSControlCharacter CC_ESC)
[1;5H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25lg
TBSpecial (TSControlCharacter CC_ESC)
[12;159H6
TBSpecial (TSControlCharacter CC_ESC)
[1;6H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25l
TBSpecial (TSControlCharacter CC_ESC)
[12;159H7
TBSpecial (TSControlCharacter CC_ESC)
[1;7H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25ls
TBSpecial (TSControlCharacter CC_ESC)
[12;159H8
TBSpecial (TSControlCharacter CC_ESC)
[1;8H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25lt
TBSpecial (TSControlCharacter CC_ESC)
[12;159H9
TBSpecial (TSControlCharacter CC_ESC)
[1;9H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25lu
TBSpecial (TSControlCharacter CC_ESC)
[12;159H10
TBSpecial (TSControlCharacter CC_ESC)
[1;10H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25lf
TBSpecial (TSControlCharacter CC_ESC)
[12;160H1
TBSpecial (TSControlCharacter CC_ESC)
[1;11H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25lf
TBSpecial (TSControlCharacter CC_ESC)
[12;160H2
TBSpecial (TSControlCharacter CC_ESC)
[1;12H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[12;1H
TBSpecial (TSControlCharacter CC_ESC)
[K
TBSpecial (TSControlCharacter CC_ESC)
[1;11H
TBSpecial (TSControlCharacter CC_ESC)
[?25l
TBSpecial (TSControlCharacter CC_ESC)
[12;147H^[
TBSpecial (TSControlCharacter CC_ESC)
[1;11H
TBSpecial (TSControlCharacter CC_ESC)
[12;147H  
TBSpecial (TSControlCharacter CC_ESC)
[1;12H
TBSpecial (TSControlCharacter CC_ESC)
[12;157H1,11
TBSpecial (TSControlCharacter CC_ESC)
[10CAll
TBSpecial (TSControlCharacter CC_ESC)
[1;11H
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[?25l
TBSpecial (TSControlCharacter CC_ESC)
[12;147H:
TBSpecial (TSControlCharacter CC_ESC)
[1;11H
TBSpecial (TSControlCharacter CC_ESC)
[12;147H
TBSpecial (TSControlCharacter CC_ESC)
[K
TBSpecial (TSControlCharacter CC_ESC)
[12;1H:
TBSpecial (TSControlCharacter CC_ESC)
[?25hq!
TBSpecial (TSControlCharacter CC_CR)
TBSpecial (TSControlCharacter CC_ESC)
[?25l
TBSpecial (TSControlCharacter CC_ESC)
[?1000l
TBSpecial (TSControlCharacter CC_ESC)
[?2004l
TBSpecial (TSControlCharacter CC_ESC)
[>4;m
TBSpecial (TSControlCharacter CC_ESC)
[23;2t
TBSpecial (TSControlCharacter CC_ESC)
[23;1t
TBSpecial (TSControlCharacter CC_ESC)
[12;1H
TBSpecial (TSControlCharacter CC_ESC)
[K
TBSpecial (TSControlCharacter CC_ESC)
[12;1H
TBSpecial (TSControlCharacter CC_ESC)
[?2004l
TBSpecial (TSControlCharacter CC_ESC)
[?1l
TBSpecial (TSControlCharacter CC_ESC)
>
TBSpecial (TSControlCharacter CC_ESC)
[?25h
TBSpecial (TSControlCharacter CC_ESC)
[>4;m
TBSpecial (TSControlCharacter CC_ESC)
[?1049l
